builder/amazon/common/access_config.go

```diff
@@ -86,3 +100,24 @@ func (c *AccessConfig) Prepare(t *packer.ConfigTemplate) []error {
 
 	return nil
 }
+
+func GetInstanceMetaData(path string) (contents []byte, err error) {
+	url := "http://169.254.169.254/latest/meta-data/" + path
```

Comment:
Yes I'd noticed the pull request and will update this accordingly when it's merged (at the moment this code doesn't retry etc which I've not sen to be an issue so far but you never know). For now that implementation is mostly lifted out of the goamz library, but with `http.Client` instead of the retrying client that library uses. @sethvargo do you want me to look more into this?